## Cohort information:

1- **Childhood SLE (cSLE) cohort**: 44 samples  

We single cell profiled ~260K peripheral blood mononuclear cells (PBMCs) from 33 children with
SLE with different degrees of disease activity and 11 matched control

2- **Childhood - adult SLE (caSLE) cohort**: 58 samples 

in addition to 33 cSLE + 11cHD, we profiled PBMC form 8 adult SLE and 6 healthy controls.

## Single cell Technology 

10X Genomics. chemistry = V2

## Scripts used for the analysis available:

- Scripts/notebooks used for the analysis 
- Data Processing using scanpy pipeline (https://scanpy-tutorials.readthedocs.io/en/latest/pbmc3k.html)
- Batch correction using BBKNN (https://github.com/Teichlab/bbknn)

## Publication associated with thess datasets:    
**Nehar-Belaid D**, Hong S, Marches R, Chen G, Bolisetty M, Baisch J, Walters L, Punaro M, Rossi R, Chung C-H, Huynh R, Singh P, Flynn W.F, Tabanor J-A, Kuchipudi N, Mejias A, Collet M, Lucido A-L, Palucka K, Robson P, Lakshminarayanan S, Ramilo O, Wright T, Pascual V and Banchereau J. Mapping SLE heterogeneity at the single cell level. Nature Immunology. 2020 Aug 3. 
[doi:10.1038/s41590-020-0743-0](https://www.nature.com/articles/s41590-020-0743-0) PMID: 32747814

## raw counts (after CellRanger processing) available in GEO: 
https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE135779
Accession number: GSE135779

## fastq  files to be requested here (dbGAP): 
https://www.ncbi.nlm.nih.gov/projects/gapprev/gap/cgi-bin/study.cgi?study_id=phs002048.v2.p1

## shiny app (Subclster analysis in cSLE): 
 https://scrnaseq-sle.jax.org/
